### ISSUE TITLE
Deprecated

### ISSUE THREAD

**Issue by githubowl** [CodetrixStudio/CapacitorGoogleAuth#231]
Hello

I just got the message that this method of google signin will soon be deprecated. Can you tell me when ypu'll be updating this please?

Thanks a lot

**Comment by Joep-DDQ** [comment]
Do you have a screenshot or more specific message? 
Seems for me the plugin stopped working today.

**Comment by githubowl** [comment]
Your client application uses libraries for user authentication or authorization that will soon be deprecated. See the [Migration Guide](https://developers.google.com/identity/gsi/web/guides/gis-migration) for more information.

It appears to still be working for me today.

**Comment by Joep-DDQ** [comment]
Yes, I think my issue was caused by trying to run the plugin on desktop (which was similated as mobile/native).

**Comment by vicatcu** [comment]
Just to be clear this _is_ an issue that will need to be addressed though, right? Because, per the link to the Migration Guide...

>Warning: The Google Sign-In JavaScript platform library for Web is [deprecated](https://developers.googleblog.com/2021/08/gsi-jsweb-deprecation.html), and unavailable for download after March 31, 2023. The solutions in this guide are based on this library and therefore also deprecated.

**Comment by felixfaure** [comment]
Hello, do you have plan to migrate to the new google auth? Thank you a lot

**Comment by NicolasRoehm** [comment]
Hi, we are 3 months away from the deprecation of the current Google Auth library. Do you have any plans ?

**Comment by MatteoOstermeier** [comment]
![image](https://user-images.githubusercontent.com/50016431/213196115-9bfbe9f8-d66d-4a62-b11b-99b633181368.png)

First of all, thanks for this plugin, which I'm very happy to use!
Same issue here. Google will shutdown support end of march 2023. Anything planned?

**Comment by rzy77** [comment]
<img width="774" alt="Screenshot 2023-01-24 at 14 32 56" src="https://user-images.githubusercontent.com/28761362/214292675-5a4a39b6-be17-438d-b6eb-4333ec0b7e06.png">

Also thanks a lot for the plugin - it is of great help. 
I'm receiving the same warnings and the changes that need to be made are found here:
[https://developers.google.com/identity/gsi/web/guides/migration#popup-mode](url)

Please let us know if there is a plan to fix this.
Thanks 👍
